king_phisher/client/assistants/campaign.py

```diff
@@ -249,6 +248,9 @@ def _get_graphql_company(self, company_name):
 		}""", {'name': company_name})
 		return results['db']['company']
 
+	def signal_toggle_time(self, _):
+		return self.time_selector.signal_button_toggled(_)
```

Comment:
I'll wait for @zeroSteiner's input because I think I remember him telling me to explicitly call it this way, the same way the `signal_assistant_apply(self, _)` is called.